package.json

```diff
@@ -45,6 +45,7 @@
     "@babel/preset-react": "^7.13.13",
     "@babel/plugin-transform-react-jsx": "^7.14.3",
     "@material-ui/core": "^4.11.4",
+    "@material-ui/icons": "^4.11.2",
```

Comment:
Don't think we want to include the entire thing, it is pretty huge. Can you put `@material-ui/core/IconButton` etc here instead? If it works

Reply:
That doesn't seem to be possible, it returns a 405.

Reply:
Ah ok. We might actually just bundle the 4 that we use, then it is fine :) Let's discuss it on planning